### PR TITLE
Fix #5200 to use new Inline algorithm for configurations of annotated tests

### LINKED ISSUE #5200: Use new Inline algorithm for configurations of annotated tests
Hi community, it is a good first issue. Welcome your attention. 😀 
Content: Modify YAML configurations with the new Inline algorithm
Difficulty rating: ★☆☆☆☆

### PR DESCRIPTION
**Pull request by JasonKing168**
Fixes #5200.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: sharding-proxy/sharding-proxy-bootstrap/src/main/resources/conf/config-sharding.yaml [hunk 1]
```diff
@@ -81,51 +90,60 @@
...
-#        column: order_item_id
-#  bindingTables:
-#    - t_order,t_order_item
-#  defaultDatabaseStrategy:
-#    inline:
-#      shardingColumn: user_id
-#      algorithmExpression: ds_${user_id % 2}
-#  defaultTableStrategy:
-#    none:
+schemaName: sharding_db
```
Comment (tristaZero) [inline sharding-proxy/sharding-proxy-bootstrap/src/main/resources/conf/config-sharding.yaml:93]: Hi, Please comment out that configuration to retain the initial status.

File: sharding-distribution/sharding-proxy-distribution/src/main/resources/conf/config-sharding.yaml [hunk 2]
```diff
@@ -80,51 +89,60 @@
...
-#        column: order_item_id
-#  bindingTables:
-#    - t_order,t_order_item
-#  defaultDatabaseStrategy:
-#    inline:
-#      shardingColumn: user_id
-#      algorithmExpression: ds_${user_id % 2}
-#  defaultTableStrategy:
-#    none:
+schemaName: sharding_db
```
Comment (tristaZero) [inline sharding-distribution/sharding-proxy-distribution/src/main/resources/conf/config-sharding.yaml:92]: Hi, Please comment out that configuration to retain the initial status.

**Comment by tristaZero** [comment]
Hi @JasonKing168 It is glad to see you again. :) BTW, could you give a look at my comment?

**Comment by JasonKing168** [comment]
> Hi @JasonKing168 It is glad to see you again. :) BTW, could you give a look at my comment?

Sure.  Review again, please, thanks.

**Comment by tristaZero** [comment]
Hi since CI test got something wrong, I merged your PR and tested Sharding Proxy 
with those new APIs. The result showed it worked well with your changes!
Thanks a lot, @JasonKing168   :-)

**Comment by tristaZero** [comment]
@JasonKing168 
 BTW, as you are the proper one to tackle with #5234, which is similar to #5200 , I'd like to listen to your voice whether you want to take charge of it? Please be free to give your opinion. 

Best,
Trista

**Pull request merged**